flycheck.el

```diff
@@ -7182,7 +7182,7 @@ See Info Node `(elisp)Byte Compilation'."
 
 Variables are taken from `flycheck-emacs-lisp-checkdoc-variables'."
   `(progn
-     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(symbol-value opt)))
+     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(custom-quote (symbol-value opt))))
```

Comment:
I meant putting the quote outside of the `,(…)`.
I don't think quoting keywords/string is an issue, is it?